.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why do we need this file?

Reply:
I don't think we need it, we can delete it.